src/main/kotlin/no/nav/syfo/sykmelding/service/EgenmeldtSykmeldingService.kt

```diff
@@ -97,9 +98,9 @@ class EgenmeldtSykmeldingService @KtorExperimentalAPI constructor(
             log.warn("Egenmeldt sykmelding kan ikke være mer enn {} dager", maxAntallDagerSykmeldt)
             throw ForLangPeriodeException("Egenmeldt sykmelding kan ikke være lenger enn $maxAntallDagerSykmeldt dager")
         }
-        if (database.sykmeldingErAlleredeRegistrertForBruker(fnr = fnr)) {
-            log.warn("Det finnes en egenmeldt sykmelding fra før for samme bruker, {}", callId)
-            throw SykmeldingAlreadyExistsException("Du kan kun benytte egenmelding én gang")
+        if (database.sykmeldingOverlapperGrenseverdi(fnr, fom, tom, 16) || database.antallSykmeldingerInnenforPeriode(fnr, fom.minusMonths(4), fom) > 1) {
+            log.warn("Man kan bare ha opp til to egenmeldt sykmeldinger ila. 4 måneder, med minimum 16 dagers opphold")
+            throw ForMangeSykmeldingerException("Du kan bare ha opp til to egenmeldt sykmeldinger ila. 4 måneder, med minimum 16 dagers opphold")
```

Comment:
Denne teksten vises til bruker, så det kan være lurt å sjekke den med Berit. Vi bør iallefall skrive "egenmeldinger" og unngå forkortelser, f.eks. slik: 
"Du kan bare ha opp til to egenmeldinger i løpet av fire måneder, med minimum 16 dagers opphold"

Reply:
Godt tenkt! Jeg sjekker med Berit :)

Reply:
Vi endret teksten og splittet reglene i to. Berit påpekte at det strengt tatt er to forskjellige tilfeller som sjekkes her. :)